app.js

```diff
@@ -47,7 +49,7 @@ function transfer(opts) {
   return referHandler.transfer();
 }
 
-class Simring {
+class Simring extends Emitter{
   constructor(req, res, uriList, opts, notifiers) {
     const callOpts = {
```

Comment:
don't you need to call `super()` as the first thing in the constructor?

Reply:
you're probably 100% right here :) Like I said - completely non tested. I'll go test the changes myself this week and we can go from there